consts.js

```diff
@@ -0,0 +1,32 @@
+// The list of the default externals
+// https://webpack.js.org/configuration/externals/
+const defaultExternals = [
+  '@folio/stripes',
+  '@folio/stripes-components',
+  '@folio/stripes-connect',
+  '@folio/stripes-core',
+  '@folio/stripes-util',
+  '@folio/stripes-form',
+  '@folio/stripes-final-form',
+  '@folio/stripes-logger',
+  '@folio/stripes-smart-components',
+  'final-form',
+  'final-form-arrays',
+  'moment',
+  'moment-timezone',
+  'react',
+  'react-dom',
+  'react-final-form',
+  'react-final-form-arrays',
+  'react-final-form-listeners',
```

Comment:
Same as `moment-timezone`; does this need to be added to the platform's direct-deps?